tests/test_landing_pad_tracking.py

```diff
@@ -211,33 +213,39 @@ def test_mark_false_positive_with_similar(self,
         Test if marking false positive adds detection to list of false positives and removes
         similar landing pads
         """
-        false_positive = object_in_world.ObjectInWorld.create(1, 1, 1)[1]
-        tracker._LandingPadTracking__unconfirmed_positives = detections2
+        _, false_positive = object_in_world.ObjectInWorld.create(1, 1, 1)
+        assert false_positive is not None
+
+        tracker._LandingPadTracking__unconfirmed_positives = detections2  # type: ignore
         expected = [false_positive]
         expected_unconfirmed_positives = [detections2[2], detections2[3], detections2[4]]
-        
+
         tracker.mark_false_positive(false_positive)
 
-        assert tracker._LandingPadTracking__false_positives == expected
-        assert tracker._LandingPadTracking__unconfirmed_positives == expected_unconfirmed_positives
-    
+        assert tracker._LandingPadTracking__false_positives == expected  # type: ignore
+        assert tracker._LandingPadTracking__unconfirmed_positives == expected_unconfirmed_positives  # type: ignore
+
     def test_mark_multiple_false_positive(self,
                                           tracker: landing_pad_tracking.LandingPadTracking,
                                           detections1: "list[object_in_world.ObjectInWorld]"):
         """
         Test if marking false positive adds detection to list of false positives
         """
-        false_positive1 = object_in_world.ObjectInWorld.create(0, 0, 1)[1]
-        false_positive2 = object_in_world.ObjectInWorld.create(2, 2, 1)[1]
-        tracker._LandingPadTracking__unconfirmed_positives = detections1
+        _, false_positive1 = object_in_world.ObjectInWorld.create(0, 0, 1)
```

Comment:
Not sure how numbers working with snake case but should we change it to `false_positive_1`?